pyfar/signals/deterministic.py

```diff
@@ -194,6 +211,102 @@ def linear_sweep_time(n_samples, frequency_range, n_fade_out=90, amplitude=1,
     return signal
 
 
+def linear_sweep_freq(
+        n_samples, frequency_range, start_margin, stop_margin, fade_in=None,
+        fade_out=None, butterworth_order=8, sampling_rate=44100):
+    """
+    Generate single channel sine sweep with linearly increasing frequency.
```

Comment:
```suggestion
    Generate single channel sine sweep with linearly increasing frequency in frequency domain.
```

Reply:
I like that the first lines are identical, because that is what the function does. The details and differences between time and frequency domain synthesis are well explained below.